website/pages/api-docs/agent/service.mdx

```diff
@@ -513,13 +513,14 @@ service definition keys for compatibility with the config file format.
 
 - `Kind` `(string: "")` - The kind of service. Defaults to "" which is a
   typical Consul service. This value may also be "connect-proxy" for
-  services that are [Connect-capable](/docs/connect)
-  proxies representing another service or "mesh-gateway" for instances of
-  a [mesh gateway](/docs/connect/mesh_gateway)
+  [Connect-capable](/docs/connect) proxies representing another service,
+  "mesh-gateway" for instances of a [mesh gateway](/docs/connect/mesh_gateway),
+  "terminating-gateway" for instances of a [terminating gateway](/docs/connect/terminating_gateway),
+  or "ingress-gateway" for instances of a [ingress gateway](/docs/connect/ingress_gateway).
 
 - `Proxy` `(Proxy: nil)` - From 1.2.3 on, specifies the configuration for a
-  Connect proxy instance. This is only valid if `Kind == "connect-proxy"` or
-  `Kind == "mesh-gateway"`. See the [Proxy documentation](/docs/connect/registration/service-registration)
+  Connect proxy instance. This is only valid if `Kind != ""`.
```

Comment:
When I was reviewing the `Kind` and `Proxy` updates, I realized the relationship between the two is a little confusing. For `Kind` you can specify a "Connect-capable" proxy and for `Proxy` you specify a "Connect proxy instance". Are we trying to say the same thing or highlight a difference with these two terms? Generally, it's confusing to just say proxy since there are many types. Can we always be specific?

Reply:
We're highlighting the difference. Connect-capable proxy is more like a category, and that's why it's used with `Kind`. On the other hand, "Connect proxy instance" is an instance of a Connect-capable proxy. The `Proxy` field is proxy-specific configuration for this service definition. 

Any thoughts on how that could be clarified?

Reply:
Just read this comment thread and was also confused by "Connect-capable" proxy. I wonder if we should just change it to `[Connect](/docs/connect) proxies representing ...`. Basically remove the `-capable` part.